staging/kos/pkg/util/parse/network/subnet/summary.go

```diff
@@ -37,6 +37,7 @@ const (
 // tasks.
 type Summary struct {
 	NamespacedName types.NamespacedName
+	UID            types.UID
```

Comment:
Subnet summary is simplified by immutability: it needs to store the UID, but some functions become useless and can be removed.